content/SCALE/GettingStarted/Install/InstallEnterpriseHASCALE.md

```diff
@@ -160,24 +161,65 @@ SCALE is now installed on controller 1 and repeated for controller 2 starting wi
 
 {{< /expand >}}
 ### Configuring the Network with Console Setup Menu
-After installing the both controller 1 and 2 <file>.iso</file> file and finishing the TrueNAS SCALE Installer process, if the TrueNAS SCALE server is connected to the network where DHCP is not enabled, use the Console setup menu to assign controller 1 main network interface the static IP address to allow access to the SCALE UI. 
-TrueNAS SCALE uses DHCP to assign an IP address to the primary network interface to allow access to the SCALE UI.
-{{< hint type=important >}}
-Only users with experience configuring network settings and using the Console setup menu should use it to configure all network settings. All other users should only use the Console setup menu to configure a static IP address for the primary network interface for controller 1 to allow access to the SCALE UI.
-The SCALE UI has safeguards in place to prevent network connectivity issues that could require a clean install of SCALE to restore access.
-{{< /hint >}}
-To use the Console setup menu to change the primary network interface IP address on controller 1:
 
-1. Type <kbd>1</kbd> and then press <kbd>Enter</kbd> to open the **Configure Network Interfaces** screen.
-2. Use either <kbd>Tab</kbd> or the arrow keys to select the interface assigned as your primary network interface.
+After installing the SCALE <file>.iso</file> file on on both controller 1 and 2 and finishing the TrueNAS SCALE Installer process, use the Console setup menu to configure the required network settings on controller 1 so it can access the SCALE UI. 
+TrueNAS SCALE single controller systems use the DHCP-assigned IP address for the primary network interface to access the SCALE UI to complete the rest of the network and other configuration settings. 
+However, HA systems with dual controllers must use static IP addresses. 
+
+To allow controller 1 to access the UI, you must disable DHCP and add the controller 1 static IP address and netmask as an alias on the primary network interface, and then enter the network settings for host name, domain name, default gateway, and the name servers (1 and 2). 
+You can configure the rest of the HA global network settings in the SCALE web UI.
+
+To use the Console setup menu to configure required network settings on controller 1:
```

Comment:
the screen title is Console setup and not Console Setup.  The screen name used to include menu, that is now gone from the screen but it is not incorrect to continue including menu. I suppose we could make this all lower case but it might get lost in text. I don't want to make all mentions in bold because it is both referred to in a general sense and when specifically talking about the screen version. I suggest leaving this as is.